mkssins.py

```diff
@@ -0,0 +1,203 @@
+#Imports
+import numpy as np
+import matplotlib.pylab as plt
+import warnings
+import time
+import pickle
+import sys
+import katcali
```

Comment:
It doesn't look any of these are needed in the module file (here). Remove them and import in the notebook instead if you need them there.